### PR TITLE
homework 0 

### PR DESCRIPTION
**Pull request by marcalassad**

### CODE REVIEW

File: people/mark-assad.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Marc Alassad
+------------------
+
+![](photos/markassad.JPG)
```
Comment (dblock) [inline people/mark-assad.md]: Please rename the JPG file to mark-assad.jpg to be consistent with the rest of the names.

File: people/mark-assad.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Marc Alassad
```
Comment (dblock) [inline people/mark-assad.md]: You're Marc Assad, right? Is there a typo here?

**Comment by dblock** [comment]
Please _squash_ your commits. Check out http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html if you need help.

**Comment by marcalassad** [comment]
I squashed all commits. Let me know if anything else is missing. Thanks!

**Comment by dblock** [comment]
Very good. You still need to _rebase_ against master and put your name in the list in alphabetical order.

**Pull request closed without merging**